ndsl/dsl/dace/orchestration.py

```diff
@@ -237,7 +237,17 @@ def _call_sdfg(dace_program: DaceProgram, sdfg: SDFG, config: DaceConfig, args,
         with DaCeProgress(config, "Run"):
             if config.is_gpu_backend():
                 _upload_to_device(list(args) + list(kwargs.values()))
-            res = config.loaded_precompiled_SDFG[dace_program]()
+
+            # NOTE: this will go over declared arguments and closure arguments.
+            # It is a very slow piece of code. Compiled SDFG comes with a "fast_call"
+            # function that expects all pointers to have been made C worthy. This is
+            # something we did with "FrozenSDFG" but we undid because it meant that
```

Comment:
That one is important because `FrozenSDFG` is a different concept in the gt4py/dace bridge. Over there it means that origin/domain were "frozen" into the SDFG (which is otherwise unfrozen).

```suggestion
            # something we did with "FrozenCompiledSDFG" but we undid because it meant that
```